packages/grid/_modules_/grid/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -53,6 +36,82 @@ export const cleanFilterItem = (
   return cleanItem;
 };
 
+const filterModelDisableMultiColumnsFilteringWarning = buildWarning(
+  [
+    'MUI: The `filterModel` can only contain a single item when the `disableMultipleColumnsFiltering` prop is set to `true`.',
+    'If you are using the community version of the `DataGrid`, this prop is always `true`.',
+  ],
+  'error',
+);
+
+const filterModelMissingItemIdWarning = buildWarning(
+  "MUI: The 'id' field is required on `filterModel.items` when you use multiple filters.",
+  'error',
+);
+
+const filterModelMissingItemOperatorWarning = buildWarning([
+  'MUI: One of your filtering item have no `operatorValue` provided.',
```

Comment:
Why it uses `console.warn` while `filterModelMissingItemIdWarning` above outputs an error?

```suggestion
  'MUI: One of the filtering items has no `operatorValue` provided.',
```

Reply:
https://github.com/mui/mui-x/pull/3671#discussion_r806837263

I kept a warning here out of consistency with the other deprecation messages.
But I can replace it with an error.

It's a param of `buildWarning` to choose the severity of the message